extensions/vscode/src/commands/stop-dev-server.ts

```diff
@@ -85,7 +85,9 @@ export const stopDevServer = async (): Promise<void> => {
 
       const stopDaemonResponse = await daemon.send(stopRequest);
       if (stopDaemonResponse.error) {
-        window.showErrorMessage(stopDaemonResponse.error.message);
+        if (stopDaemonResponse.error.message !== "Application not found") {
```

Comment:
I wonder if we could have a less fragile way of checking this, `Application not found` is more like a label than a code, so if the casing or wording change, this might fail.

I know this might need changes on the daemon, so probably not something that can be solved on this PR.

In either way, a small improvement would be to compare this ignoring the casing at least.